x-pack/plugins/security_solution/public/assistant/get_comments/stream/stream_observable.ts

```diff
@@ -39,24 +58,72 @@ export const getStreamObservable = (
               observer.complete();
               return;
             }
+            const codec = new EventStreamCodec(toUtf8, fromUtf8);
```

Comment:
I could not find documentation on how to handle their codec. I did find [this issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/5541) for when documentation should be added.

I followed the bedrock extension in the langchainjs repo here: https://github.com/langchain-ai/langchainjs/pull/2367/files#diff-ead6be41360b3bf1c4559ea4d0069a015b69a57d5f8c51d0cce73253fde30aa2R246-R268